src/status_im/contexts/onboarding/intro/view.cljs

```diff
@@ -2,53 +2,49 @@
   (:require
     [quo.core :as quo]
     [react-native.core :as rn]
-    [status-im.constants :as constants]
     [status-im.contexts.onboarding.common.background.view :as background]
     [status-im.contexts.onboarding.common.overlay.view :as overlay]
     [status-im.contexts.onboarding.intro.style :as style]
     [status-im.contexts.onboarding.terms.view :as terms]
-    [status-im.contexts.syncing.scan-sync-code.view :as scan-sync-code]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
 (defn view
   []
   [rn/view {:style style/page-container}
    [background/view false]
-   [quo/drawer-buttons
-    {:on-init             (fn [reset-top-animation-fn]
-                            (reset! scan-sync-code/dismiss-animations reset-top-animation-fn))
-     :animations-duration constants/onboarding-modal-animation-duration
-     :animations-delay    constants/onboarding-modal-animation-delay
-     :top-card            {:on-press            #(rf/dispatch [:open-modal
-                                                               :screen/onboarding.sign-in-intro])
-                           :heading             (i18n/label :t/sign-in)
-                           :animated-heading    (i18n/label :t/sign-in-by-syncing)
-                           :accessibility-label :already-use-status-button}
-     :bottom-card         {:on-press            (fn []
-                                                  (when-let [blur-show-fn @overlay/blur-show-fn-atom]
-                                                    (blur-show-fn))
-                                                  (rf/dispatch
-                                                   [:open-modal :screen/onboarding.new-to-status]))
-                           :heading             (i18n/label :t/new-to-status)
-                           :accessibility-label :new-to-status-button}}
-    [quo/text
-     {:size   :paragraph-2
-      :weight :regular
-      :style  style/plain-text}
-     (i18n/label :t/you-already-use-status)]
-    [quo/text {:style style/text-container}
-     [quo/text
-      {:size   :paragraph-2
-       :weight :regular
-       :style  style/plain-text}
-      (i18n/label :t/by-continuing-you-accept)]
-     [quo/text
-      {:on-press #(rf/dispatch [:show-bottom-sheet
-                                {:content (fn [] [terms/terms-of-use])
-                                 :shell?  true}])
-       :size     :paragraph-2
-       :weight   :regular
-       :style    style/highlighted-text}
-      (i18n/label :t/terms-of-service)]]]
+   [quo/bottom-actions
+    {:container-style  {:position         :absolute
+                        :width            "100%"
```

Comment:
% shouldn't be used

Reply:
what should I use instead?

Reply:
it should just work without :width

Reply:
it's not at full width when no width 100%
![CleanShot 2024-06-13 at 16 02 31](https://github.com/status-im/status-mobile/assets/15090582/40170a8d-11e6-47fc-aaaa-8dfdda202083)

Reply:
@yqrashawn, what I believe @flexsurfer means is that most of the time when percentages are necessary is because the underlying implementation is incorrect. Here I see the bottom actions should be able to adjust to screen width using flex rules and no percentages. This could be a bug in some component and not in your usage of it, in which case it'd be a good idea to try to fix it either in this PR if it's simple and safe or in a separate PR.

The exception I've seen to use percentages is when dealing with image dimensions, but here it's not the case.

Reply:
yes, thank you for the explanation Icaro

Reply:
But in this case, how can I reach full width when the container is `absolute`? Use `:dimensions/window-width` or set both left and right to 0?

Reply:
I updated the code to use `:left 0 :right 0`